src/detector/implementations/picamera2_implementation.py

```diff
@@ -0,0 +1,181 @@
+from src.detector.api import \
+    GetCaptureDeviceResponse, \
+    GetCaptureImageRequest, \
+    GetCaptureImageResponse, \
+    GetCapturePropertiesResponse, \
+    SetCaptureDeviceRequest, \
+    SetCapturePropertiesRequest
+from src.common import \
+    EmptyResponse, \
+    ErrorResponse, \
+    get_kwarg, \
+    MCastResponse
+from src.common.structures.capture_status import CaptureStatus
+
+from src.detector.implementations import AbstractCameraInterface
+
+from picamera2 import Picamera2, Preview
```

Comment:
Wouldn't this add a new requirement to the project?

Reply:
Picamera has to be installed with apt, not pip in my experience

Reply:
Would I be correct in thinking that this dependency will not be resolvable on Windows?

Reply:
If so, then perhaps the comment above will be important to address before merge

https://github.com/PerkLab/MCSTrack/pull/30#discussion_r1616316556